pkg/operator/ceph/config/keyring/cephx.go

```diff
@@ -28,12 +28,17 @@ import (
 
 var CephAuthRotateSupportedVersion = version.CephVersion{Major: 20, Minor: 2, Extra: 0}
 
-// CephxKeyIdentifierAnnotation is the annotation that should be applied to pod specs to
-// ensure that pods restart after keys are rotated (and not restarted when keys are not rotated).
-// The keyring secret resourceVersion is suggested but not always available.
-//
-//nolint:gosec // G101: this is not hardcoded credentials
-const CephxKeyIdentifierAnnotation = "cephx-key-identifier"
+const (
+	// CephxKeyIdentifierAnnotation is the annotation that should be applied to pod specs to
+	// ensure that pods restart after keys are rotated (and not restarted when keys are not rotated).
+	// The keyring secret resourceVersion is suggested but not always available.
+	//
+	//nolint:gosec // G101: this is not hardcoded credentials
+	CephxKeyIdentifierAnnotation = "cephx-key-identifier"
+
+	// RBDMirrorPeerCephx represents the `rbd-mirror-peer` user
+	RBDMirrorPeerCephx = "rbd-mirror-peer"
```

Comment:
This is already captured here: https://github.com/rook/rook/blob/ba6f8480d6bf110624c4ccdb1c5e4c46b08f4dc6/pkg/daemon/ceph/client/mirror.go#L62

Could we combine these somehow, avoiding import loops?